nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/io/TaskWriter.java

```diff
@@ -115,6 +115,7 @@ public void writeEntityBody(Task item, JsonGenerator jg) throws IOException {
             jg.writeStringField("id", item.getDocument().getId());
             jg.writeStringField("name", item.getName());
             jg.writeStringField("workflowInstanceId", workflowInstanceId);
+            jg.writeBooleanField("allowTaskReassignment", node != null && node.allowTaskReassignment());
```

Comment:
Could be inside this [section](https://github.com/nuxeo/nuxeo-platform-document-routing/pull/91/commits/ee0d2b259d04f571f80bdefc35f785224024d0b3#diff-3bffd5f6e4082014bee90bed7a055a1bR182) where `node` is already assumed to be not null (detail).